### ISSUE TITLE
[cygnus-ngsi] Notification Error

### ISSUE THREAD

**Issue by oliveiradrodrigo** [telefonicaid/fiware-cygnus#1861]
I have recently updated Cygnus version from 1.9.0 to 2.0.0 because of Mongo Auth Source environment variable but I'm running into an error saying that the response from Cygnus is empty (no headers, no data).

![Screenshot from 2020-04-23 19-35-18](https://user-images.githubusercontent.com/55299176/80156191-aef4f100-8599-11ea-8176-2474b3156737.png)

![WhatsApp Image 2020-04-23 at 19 12 23](https://user-images.githubusercontent.com/55299176/80156287-ec597e80-8599-11ea-81ad-234d3f932dc3.jpeg)

I've made a subscription in Orion using the following configuration:
```
{
  "description": "Notify Cygnus of all context changes",
  "subject": {
    "entities": [
      {
        "idPattern": ".*"
      }
    ]
  },
  "notification": {
    "http": {
      "url": "http://192.168.0.9:5050/notify"
    },
    "attrsFormat": "legacy"
  },
  "throttling": 5
}
```

In version 1.9.0 the notification works just fine but I need Mongo Auth Source var to work with.

Is there any advice or solution incoming to this issue?

Thanks in advance! :)

**Comment by fgalan** [comment]
Which Cygnus configuration are you using (the one that works in Cygnus 1.9.0 but not in 2.0.0)?

Thanks for the feedback!

**Comment by oliveiradrodrigo** [comment]
Yes, @fgalan ! I was using the 1.9.0 and I saw that the team created the environment var for Mongo Auth Source that I need to use but my Orion can't communicate with Cygnus since (I think) there is no support for NGSIv2 in version 2.0.0.

I also tried version 2.1.0 but the error was not fixed in this version too.

Thanks for the attention! :)

**Comment by fgalan** [comment]
NGSIv2 notifications support was introduced y Cygnus 1.18.0 and should work in any later version (including 1.9.0, 2.0.0 and 2.1.0).

Which Cygnus configuration are you using (the one that works in Cygnus 1.9.0 but not in 2.0.0)? I mean the .conf files. Could you include them in a comment on this issue? Obfuscating passwords, IPs, and sensible information, if any.

Thanks!
